test/gui/shared/steps/steps.py

```diff
@@ -65,6 +88,17 @@ def step(context, username, password):
     except LookupError:
         pass
 
+@Given('user "|any|" has set up a client with poll interval settings and password "|any|"')
```

Comment:
I think we can use a step where you can provide any poll interval value from here rather than using the default 5000

Reply:
Done